Library/Formula/qt-gstreamer.rb

```diff
@@ -0,0 +1,66 @@
+class QtGstreamer < Formula
+  desc "C++ bindings for GStreamer with a Qt-style API"
+  homepage "http://gstreamer.freedesktop.org/modules/qt-gstreamer.html"
+  url "http://gstreamer.freedesktop.org/src/qt-gstreamer/qt-gstreamer-1.2.0.tar.xz"
+  sha256 "9f3b492b74cad9be918e4c4db96df48dab9c012f2ae5667f438b64a4d92e8fd4"
+  head "git://anongit.freedesktop.org/gstreamer/qt-gstreamer"
+
+  option "with-static", "Build QtGStreamer as a static library"
+  option "with-tests", "Build QtGStreamer's tests"
+  option "without-examples", "Build QtGstreamer's examples"
+  option "with-codegen", "Build and use QtGStreamer's codegen"
```

Comment:
I chose the same defaults for build options that QtGStreamer has. Usually you don't need codegen. AFAIK, it rebuilds something that is already in the repository.